demo-ui/app/uk/gov/ons/addressIndex/demoui/views/MatchTypeHelper.scala

```diff
@@ -4,7 +4,7 @@ object MatchTypeHelper {
 
   def matchType(addressId: String, ids: Seq[String], matchedFormattedAddress: String): String = {
     val matchedIds = ids.count(_ == addressId)
-    if(matchedIds == 1) {
+    if(matchedIds == 1 && !matchedFormattedAddress.isEmpty) {
```

Comment:
`!matchedFormattedAddress.isEmpty` is equivalent to `matchedFormattedAddress.nonEmpty`